contracts/test-helpers/ShareableMock.sol

```diff
@@ -0,0 +1,19 @@
+pragma solidity ^0.4.4;
+import "../Shareable.sol";
+
+contract ShareableMock is Shareable {
+
+  uint public count = 0;
+
+  function ShareableMock(address[] _owners, uint _required) Shareable(_owners, _required) {
+
+  }
+
+  function increaseCount(bytes32 action) onlymanyowners(action) {
+    count = count + 1;
+  }
+
+  function isOwnerConst(address _addr) constant returns (bool) {
+    return isOwner(_addr);
+  }
```

Comment:
Calling isOwner() in web3js does not return the function's return value because it's not const. Instead, it returns the transaction hash. Adding the isOwnerConst() function to the mock contract allows me to get around that.